File: AppGenerateFiles/helpdesk/Model/CaseTable_Comments.cs
using CAPA_DATOS;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
namespace DataBaseModel {
   public class CaseTable_Comments : EntityClass {
       [PrimaryKey(Identity = true)]
       public int? Id_Comentario { get; set; }
       public string? Body { get; set; }
       public string? Estado { get; set; }
       public DateTime? Fecha { get; set; }
       public int? Id_Case { get; set; }
       public int? Id_User { get; set; }
       public string? NickName { get; set; }
       public string? Mail { get; set; }
       public string? Attach_Files { get; set; }
       public string? Mails { get; set; }
       [ManyToOne(TableName = "CaseTable_Case", KeyColumn = "Id_Case", ForeignKeyColumn = "Id_Case")]
       public CaseTable_Case? CaseTable_Case { get; set; }
   }
}

File: AppGenerateFiles/helpdesk/FrontModel/CaseTable_Comments.js
//@ts-check
import { EntityClass } from "../../WDevCore/WModules/EntityClass.js";
import { WAjaxTools } from "../../WDevCore/WModules/WComponentsTools.js";
//@ts-ignore
import { ModelProperty } from "../../WDevCore/WModules/CommonModel.js";
import { CaseTable_Case }  from './CaseTable_Case.js'
class CaseTable_Comments extends EntityClass {
   constructor(props) {
       super(props, 'EntityHelpdesk');
       for (const prop in props) {
           this[prop] = props[prop];
       }
   }
   /**@type {Number}*/ Id_Comentario;
   /**@type {String}*/ Body;
   /**@type {String}*/ Estado;
   /**@type {Date}*/ Fecha;
   /**@type {Number}*/ Id_User;
   /**@type {String}*/ NickName;
   /**@type {String}*/ Mail;
   /**@type {String}*/ Attach_Files;
   /**@type {String}*/ Mails;
   /**@type {CaseTable_Case} ManyToOne*/ CaseTable_Case;
}
export { CaseTable_Comments }
